### PR TITLE
139: Redesigned transaction history

### LINKED ISSUE #139: Transaction History UI Update
**Description:**

The transaction history component gets clunky when there's a lot of transactions. We need a better way of representing the transaction history, so move it to a new page and update its design.

Inspo: image_123986672

### PR DESCRIPTION
**Pull request by ZackHolmberg**
image

closes #139

### CODE REVIEW

File: src/frontend/src/pages/TransactionHistoryPage.vue [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.transaction-history-header {
+  font-size: $default-header-font-size;
+  text-align: center;
+  font-weight: $transaction-history-header-font-weight;
+  color: $default-text-color;
+}
+
+.transaction-history-back-button {
```
Comment (madisonfines) [inline src/frontend/src/pages/TransactionHistoryPage.vue:72]: Verify here if the back button only activates when you hit the button and not around it!

File: src/frontend/src/style.scss [hunk 2]
```diff
@@ -141,15 +141,39 @@ $account-label-padding-top: 20px;
...
+$no-transactions-text-color: gray;
 
 $content-padding: 20px;
-$content-background-color: rgb(231, 231, 231);
+$content-background-color: black;
 $content-border-radius: 20px;
-$content-font-size: 20px;
+$content-height: 650px;
+
+
```
Comment (madisonfines) [inline src/frontend/src/style.scss]: Extra space!

File: src/frontend/src/style.scss [hunk 3]
```diff
@@ -141,15 +141,39 @@ $account-label-padding-top: 20px;
...
+$row-amount-padding: 5px;
+$row-amount-margin-top: 5%;
+
+$list-row-border-bottom: 1px solid white;
+$list-row-color: white;
+$list-row-background-color: black;
+$list-row-padding: 20px;
+
+$transaction-history-back-button-margin-top: 25px;
+$transaction-history-header-font-weight: bold;
```
Comment (madisonfines) [inline src/frontend/src/style.scss]: Maybe create a default font weight that is bold and can be used wherever we bold the font

**Review by madisonfines — CHANGES_REQUESTED** [review]

File: src/frontend/src/style.scss (hunk 3 above)
Comment (ZackHolmberg) [reply src/frontend/src/style.scss]: Now that you point it out it's kind've redundant to do this as "bold" is already an abstracted value. I'll just use bold in the component instead

File: src/frontend/src/pages/TransactionHistoryPage.vue (hunk 1 above)
Comment (ZackHolmberg) [reply src/frontend/src/pages/TransactionHistoryPage.vue:72]: Yep it's good 👍

**Review by abhisachdev17 — APPROVED** [review]
That looks so fancy! Nice work, one nit pick can we change "send" to "sent" and "receive" to "received"?

**Review by madisonfines — APPROVED** [review]
Nice job

**Pull request merged**